lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -64,6 +64,7 @@ def _namespace_has_repo_name(namespace, repo_name, resolution_cache):
     if resolution_cache is not None and cache_key in resolution_cache:
         repo_names = resolution_cache.get(cache_key)
     else:
+        log.debug("Resolution cache miss: " + cache_key)
         next_page = None
         repo_names = []
```

Comment:
```suggestion
    repo_names = []
    if resolution_cache is not None:
        if cache_key in resolution_cache:
            repo_names = resolution_cache["cache_key"]
        else:
            log.debug("Resolution cache miss: %s", cache_key)
    if not repo_names:
        next_page = None
```

Reply:
Perhaps I should change the wording of the log output. Your proposed change will not emit a log entry if resolution_cache is None. I wanted to log any time this expensive query was occurring.

See #13619

Reply:
Yep, if `resolution_cache` is None, it's not really a cache miss :)

I'd tend to suggest closing this PR and concentrate on #13619 then.